src/test/java/tests/BaseClass.java

```diff
@@ -8,12 +8,14 @@
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Optional;
 import org.testng.annotations.Parameters;
+import org.testng.asserts.SoftAssert;
 
 import java.util.logging.Logger;
 
 public class BaseClass {
     protected WebDriver driver;
-    Logger logger = Logger.getGlobal();
+    protected Logger logger = Logger.getGlobal();
+    SoftAssert softAssert = new SoftAssert();
```

Comment:
also protected?

Reply:
Thank you, I added protected access modifier. Please see changes In the following commit.

I use SoftAssert because one NavigationTest includes number of asserts (4 times verifying 4 different pages) so it is logical, in my opinion, to check all items even if the first assert fails.
Is it bad practice to use SoftAssert and I should refactor it?

Reply:
For your navigation test - it is ok :)